frontend/src/components/Dashboard.js

```diff
@@ -7,16 +7,16 @@ import { useTable } from 'react-table';
 import Table from '@material-ui/core/Table';
 import { withRouter } from 'react-router-dom';
 import Paper from '@material-ui/core/Paper';
-import CreateProjectDialog from './CreateProjectDialog';
+import ProjectDialog from './ProjectDialog';
+import InfoOutlinedIcon from '@material-ui/icons/InfoOutlined';
 
 import AddProjectBtn from './layout/AddProjectBtn';
 import './style/Dashboard.css';
 // this function will be an axios call to one of the routes
 // TODO: what is the max number of documents that the database will return?
```

Comment:
for merge, can u remove all these TODO comments and replace them with comments regarding the functions purpose?